SpecializedServers/MultiSocks/static/EA/ME3_CONFIG/ME3DATA - Copy.txt

```diff
@@ -0,0 +1,16 @@
+GAW_SERVER_BASE_URL ; http://waleu2.tools.gos.ea.com/wal/masseffect-gaw-pc
```

Comment:
Maybe an accidental copy left?

Reply:
I will remove, but this was for PC variant

Reply:
Ah! Then I think it could be nice to see if there can be ps3/pc/xbox detectiob perhaps?

Reply:
If you think it would be beneficial sure.

Reply:
It is optional really, ideally ps3 comes first.

Reply:
I'll leave it renamed for now in static.